superx/templates/home.html

```diff
@@ -43,7 +43,14 @@ <h1 id="header" class="display-3 text-center">superX</h1>
         {% else %}
             <td>{{ product.quantity|int }} {{ product.unit_of_measure }}</td>
         {% endif %}
-       <td> <button onclick="addItem({{ product.id }}, '{{product.name |replace('\'', '')}}')" type="button" class="btn btn-outline-success">הוסף לעגלה</button> </td>
+        <td>
+        <a href=# id=addItem></a>
+        <form action="{{ url_for('addItem') }}" method="post">
+          <input type="hidden" name="pruduct_id" value="{{product.id}}">
+          <button onclick="addItem({{ product.id }}, '{{product.name |replace('\'', '')}}')" type="submit" class="btn btn-outline-success">הוסף לעגלה</button> 
+      </form>
+    </a>
+    </td>
```

Comment:
This method will send the product.id when the customer will pick it from the list.
We want to let him the option to delete some of his products that already in the basket. (with the "הסר מהעגלה" button)
What I did is to send a list with items code that presents in "carttable" and only when the customer is pressing the "השווה מחירים" button.